src/components/shared/DropdownProfile.tsx

```diff
@@ -36,11 +38,20 @@ export default function DropdownProfile() {
     }
   }, [user]);
 
-  const handleLogout = () => {
-    dispatch(clearUser());
-    Cookies.remove("userType");
-    Cookies.remove("token");
-    router.replace("/auth/login");
+  const handleLogout = async () => {
+    try {
+      
+      await signOut(auth);// Log out from Firebase
```

Comment:
Move this function to the `firebaseUtils.tsx` file, where the login function is also located. This will help avoid importing `auth` directly in the initial file.

Reply:
Sure sir . I will make the change.

Reply:
**@Akhil-chaudhary**  sir, I have made the changes. Please check if they are correct.